org.mixedrealitytoolkit.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -161,6 +161,32 @@ public InteractionFlags AllowedInteractionTypes
             set => allowedInteractionTypes = value;
         }
 
+        [SerializeField]
+        [Tooltip("The movement type of the rigid body during manipulation.")]
+        private MovementType rigidbodyMovementType = MovementType.VelocityTracking;
+
+        /// <summary>
+        /// The <see cref="XRBaseInteractable.MovementType"/> of the rigid body during manipulation.
+        /// </summary>
+        /// <remarks>
+        /// This is the default <see cref="XRBaseInteractable.MovementType"/> of the rigid body during manipulation,
+        /// however, the type can be overridden by an interactor with <see cref="XRBaseInteractor.selectedInteractableMovementTypeOverride"/>.
+        /// </remarks>
+        public MovementType RigidbodyMovementType
+        {
+            get => rigidbodyMovementType;
+            set
+            {
+                rigidbodyMovementType = value;
+
+                if (isSelected && rigidBody != null)
+                {
+                    UpdateRigidbodyMovementType();
```

Comment:
Should we check if the value has actually changed here, to avoid unnecessary work?

Reply:
Good point, done.